tests/integration/states/test_folder.py

```diff
@@ -87,3 +87,4 @@ def test_move_folder(patch_salt_globals_folder_state):
     assert ret["result"] == True
     assert ret["comment"] == "moved"
     assert ret["changes"]["new"] == "folder test_folder_state_new moved to top_folder"
+    folder.manage("top_folder", "destroy", "Datacenter", "vm")
```

Comment:
**suggestion** should this go into a fixture instead, so regardless of assertion it will remove the folder?